client/src/app.routing.js

```diff
@@ -89,6 +89,15 @@ const routing = function ($stateProvider, $urlRouterProvider, $locationProvider)
     }
   };
 
+  const plannerIdeaState = {
```

Comment:
add //TODO: update to /idea/:ideaId once event service wired up, include update to ui-sref in ideas-card